app/_hub/kong-inc/proxy-cache-advanced/index.md

```diff
@@ -152,6 +152,12 @@ params:
       value_in_examples:
       description: |
         Unhandled errors while trying to retrieve a cache entry (such as redis down) are resolved with `Bypass`, with the request going upstream.
+  extra: |
+
+    **Warning:** The `content_type` requires an exact match. For example, if your Upstream expects
+    `application/json; charset=utf-8` and the `config.content_type` value is only `application/json`
+    (a partial match), then the proxy cache is bypassed.
```

Comment:
```suggestion
    <div class="alert alert-ee red">
    <strong>Warning:</strong> The <code>content_type</code> parameter requires
    an exact match. For example, if your Upstream expects
    <code>application/json; charset=utf-8</code> and the
    <code>config.content_type</code> value is only <code>application/json</code>
    (a partial match), then the proxy cache is bypassed.
    </div>
```
You can use html in these pipe extended descriptions with no problems.

Reply:
> You can use html in these pipe extended descriptions with no problems.

I was having problems with it:  `[16:48:32]              Error: YAML Exception reading /Users/terryblessing/github/docs.konghq.com/app/_hub/kong-inc/proxy-cache/index.md: (<unknown>): could not find expected ':' while scanning a simple key at line 102 column 1`

I'll try committing your suggestions.

Reply:
@lena-larionova it worked but with the highlighting changes underway, it looks a bit odd. Can we use red text within these warnings only?

![image](https://user-images.githubusercontent.com/3756245/97188646-be840b00-1771-11eb-9645-4699d14dd2e4.png)

Reply:
Sure, I can add it to the alerts overhaul ticket: https://konghq.atlassian.net/browse/DOCS-1379